shared/src/simulation/activities/index.ts

```diff
@@ -0,0 +1,19 @@
+import { unloadVehicleActivity } from './unload-vehicle';
+
+export const simulationActivities = {
+    unloadVehicleActivity,
+};
+
+export type ExerciseSimulationActivity = typeof unloadVehicleActivity;
```

Comment:
I guess this should be defined similar to the `ExerciseSimulationBehavior` (i.e. defining `ExerciseSimulationActivity` depending on `simulationActivities`).